content/developer/reference/javascript/qweb.rst

```diff
@@ -3,9 +3,9 @@
 
 .. _reference/qweb:
 
-====
-QWeb
-====
+==========
+QWeb Views
+==========
```

Comment:
Yes, but I'd like to avoid having a title "QWeb"; for someone learning Odoo he might not know what QWeb is.
I renamed into "QWeb Template", it's more accurate.